### ISSUE TITLE
April 2015 Node Breakfast

### ISSUE THREAD

**Issue by subfuzion** [BayNode/nodebreakfast#5]
We haven't had a Node Breakfast in ages? Anybody up for one in the near future?  io.js brunch?  ;-)

**Where:** Polyvore, Mountain View ([map](https://www.google.com/maps/place/Polyvore/@37.393703,-122.076619,17z/data=!3m1!4b1!4m2!3m1!1s0x808fb7314a9314cb:0x1602ddcf1b45253))

**When:** 9 - 10:30 AM, April 24, 2015

**Who (max RSVPs: 25)**:
- [ ] Adam Crabtree  @CrabDude (Noderiety)
- [x] Alex Liu @DonutEspresso (Neflix)
- [x] Aaron Heckmann @aheckmann (Pebble)
- [ ] Bert Belder @piscisaureus (StrongLoop)
- [x] Chanda Dharap @chandadharap (StrongLoop)
- [ ] Dav Glass @davglass (Yahoo)
- [x] Daniel Shaw @dshaw (NodeSource) 
- [ ] Erik Toth @totherik (PayPal)
- [x] Fred K. Schott @FredKSchott  (Worklife)
- [x] Greg Stock @GregStock (Polyvore)
- [x] Henry Allen-Tilford @generalhenry (Medallia)
- [x] Jacob Chapel @chapel (Walmart)
- [ ] Jeff Harrell @jeffharrell (PayPal)
- [ ] Micah Ransdell @micahr (Netflix)
- [ ] Nicholas Zakas @nzakas (Box)
- [x] Reid Burke @reid (Yahoo)
- [x] Robert Schultz @robertschultz (Ancestry)
- [x] Ross Kukulinski @rosskukulinski (Yodlr.io)
- [x] Tony Pujals @tonypujals (Codefresh)
- [x] Yunong Xiao @yunong (Netflix)

**Comment by subfuzion** [comment]
Missing anyone we want to include in the list?

**Comment by DonutEspresso** [comment]
Bummer - don't think I can make either 4/10 or 4/17.

**Comment by subfuzion** [comment]
How about 4/24?  And doesn't have to be a Friday ... feel free to propose a date that works!

**Comment by subfuzion** [comment]
I propose Specialtys in Mountain View ... easy freeway access, next to Hacker Dojo...

http://www.yelp.com/biz/specialtys-cafe-and-bakery-mountain-view

I can get there early and grab a big table or the back area ...

**Comment by subfuzion** [comment]
I missed the one at Hobee's ... how did that work out? That also sounds pretty good.

http://www.yelp.com/biz/hobees-restaurant-mountain-view

Also there is Clarke's on El Camino with easy access from 85 and 237:

http://www.yelp.com/biz/clarkes-charcoal-broiler-mountain-view

Not just good burgers, they have pretty good breakfast, too. Great outdoor seating if the weather's good.

**Comment by chapel** [comment]
I am fine with most dates, though this week is short notice.

Next week or the week after would work. Thursday or Friday works best.

**Comment by yunong** [comment]
Most dates work for me -- I just need a few days of heads up.

-Y

**Comment by aheckmann** [comment]
I'm not on the list but ether day SGTM.

**Comment by subfuzion** [comment]
@aheckmann You are now :)

**Comment by subfuzion** [comment]
Try voting on the best dates for you [here](http://whenisgood.net/nodebreakfast-april-2015)

And where to eat [here](http://www.tricider.com/admin/3a9N987Nbtp/3RJNdhjTFs1)

**Comment by DonutEspresso** [comment]
4/24 works! Also voted for a bunch of other dates. Venue is flexible - I'll go anywhere.

**Comment by FredKSchott** [comment]
Awesome, it's been way too long :) I'm no longer working in the South Bay, but I can definitely make it down for a morning.

**Comment by reid** [comment]
Voted. April 20 or after is good for me...

**Comment by subfuzion** [comment]
@FredKSchott You working up in the city now?

**Comment by subfuzion** [comment]
Results of the date vote so far:

http://whenisgood.net/nodebreakfast-april-2015/results/mbn52e8

**Comment by subfuzion** [comment]
Only one vote for a venue so far:
http://www.tricider.com/admin/3a9N987Nbtp/3RJNdhjTFs1

**Comment by nzakas** [comment]
Sadly, still too sick to be out in public. Have fun!

**Comment by trevnorris** [comment]
Sorry. I've moved out of CA. Won't be able to attend.

**Comment by subfuzion** [comment]
I am really sorry to hear that, @nzakas. I have a friend going through the same thing. I wish you the best, man.

**Comment by subfuzion** [comment]
@trevnorris forgot about that! hope you're enjoying life in utah. Well, hopefully @dshaw will provide some NodeSource representation! :+1:

**Comment by dshaw** [comment]
Voted!

**Comment by subfuzion** [comment]
:facepunch:

**Comment by rosskukulinski** [comment]
Voted as well.  I'll be in SF April 16th & 17th for Container.camp if anyone wants to grab coffee/drink.

**Comment by subfuzion** [comment]
@rosskukulinski I will be happy to join you for coffee/drink. Looking forward to your presentation. :+1:

**Comment by subfuzion** [comment]
Hey everyone ... well picking a day was challenging. It looks like the best day will be April 24, although unfortunately that means @rosskukulinski and @micahr are out. I'm going to set the date as April 24 based on the vote, but feel free to chime in if there's a compelling reason to pick another day...

**Comment by subfuzion** [comment]
The only vote on a venue so far was mine (specialtys). However, Greg Stock at Polyvore has offered the use of their office. They are located right by the Mountain View caltrain, a block away from Castro street. They would like to sponsor, so what do you all think?

**Comment by reid** [comment]
ok with me!

**Comment by chapel** [comment]
I'm cool with that.

**Comment by subfuzion** [comment]
Hey everyone, there is one more thing we need to address here. Can you all think of your network and make recommendations on how we can be more inclusive? There are only men in this list; surely between all of us we can invite women in the node community at some of the larger companies or enterprise companies. We've grown, but this is looking embarrassingly like a men-only club...

**Comment by chapel** [comment]
@tonypujals you make a good point and we should reach out and see if there is anyone at all interested. Hitting up twitter now.

**Comment by subfuzion** [comment]
@chapel ... that's great. I think the issue is that this has been a limited invitation-only group for individuals that are heavily involved in node performance, large scale deployment, and enterprise issues. We don't actually have a charter (we should fix that) and we need to be conscious of being as inclusive as possible within the node leadership community.

**Comment by subfuzion** [comment]
I suspect that @dshaw and @rosskukulinski can really help in this area, but ideally we should all be doing what we can.

**Comment by affinity-matrix** [comment]
Hey _guys_,

I got a ping from @chapel on Twitter! I'm happy to share my thoughts on the matter:

I think it's commendable of @tonypujals and @chapel to attempt to address the issue of inclusiveness with regards to invitees. With that said, I'm not convinced this is the best way to go about it. I can understand the sentiment behind wanting to foster a more diverse environment for your meet-up, but an approach this direct leaves me feeling a bit uneasy...

I have already been put in several situations where it becomes apparent to me that I have been invited so that I may serve as a token representative for a certain subculture or demographic. While I'm sure that isn't the motivation here, it doesn't save me from feeling slightly apprehensive about the idea of suddenly being invited to a breakfast club that has existed for about a year as a men's-only event (_even if by accident_).

I appreciate @tonypujals and his concern for cultivating inclusivity, but I would have felt a lot more comfortable with an invitation if it hadn't been framed from this angle. 

**tl;dr** — I think the question you really need to ask is, _"how can I become generally less exclusive in my everyday social habits?"_, rather than _"how can we get more women to come have breakfast with us?"_

Maybe this is a subject that could be discussed at your next breakfast!

Thanks for reading,
— Emily Rose

**Comment by chapel** [comment]
Thank you for your thoughts Emily. I didn't think about how uncomfortable
the approach would make you or anyone else. I am very sorry for not
considering you or anyone's feelings on this matter.

I hope the bluntness of my approach doesn't cause further exclusion.

**Comment by affinity-matrix** [comment]
@chapel I know you well enough to know that you meant well by soliciting my thoughts. That's the main reason I took the effort to write an honest take from my point of view.

It's great that you're willing to talk about this, and I think you should encourage the rest of the guys to talk about it as well!

**Comment by subfuzion** [comment]
Hi @emilyrose, yes I agree. If you look at my original comments here, it was to start a conversation about being more inclusive, not to fire off invitations arbitrarily just to put a bandaid on the imbalance. It was a call to do better. @chapel might have jumped the gun a bit, but I think it was done in good faith. You make a good point about changing behavior from the get-go and this definitely should be a discussion topic and self-improvement opportunity. When I saw the tweet I immediately sent a dm to the effect that we want to be inclusive, but not just invite anyone arbitrarily just to hit some numbers. Anyway, at this point it's about making a genuiine effort to do better and all advice and constructive criticism is sincerely welcome.

Any advice about moving forward in an appropriate manner with a credible charter and invitation criteria, etc, is especially appreciated.

**Comment by subfuzion** [comment]
I'm at the gym painfully typing on my phone. In any case, if the message isn't perfect, the intention to do the right thing is sincere. There has never been any overt desire to be exclusive; you are absolutely right to challenge all of us to change mindsets from the beginning to ensure inclusiveness.

**Comment by dshaw** [comment]
Thanks for the feedback, @emilyrose. Always appreciated.

I'm happy to help node breakfast evolve to the next level if there's critical mass. I'm unfortunately traveling constantly lately, so I cannot personally prioritize driving this at this time.

**Comment by subfuzion** [comment]
Hey folks, I just wanted to introduce Chanda Dharap to the group. I helped mentor one of the Node teams at Women Who Code Silicon Valley last week and she was there. She leads the engineering team at StrongLoop, so having her join us at breakfast is a no-brainer. Looking forward to Friday. Please welcome her!  :+1:

**Comment by rosskukulinski** [comment]
Hi Chanda Dharap - looking forward to meeting you.

**Comment by CrabDude** [comment]
Welcome @chandadharap! Glad to have you join us. I hope to see you at breakfast.

Another option for breakfast is [Baji's](http://bajis.com) in Mountain View. They should be dead around that time, and if I let them know, we could probably setup a large table in the back-room. Of course, it's a little less drop-in friendly, but they're crazy nice, cheap and tasty. Just bring cash and I'll provide change & cover the difference.

My only worry about Specialty's is that it'll probably get pretty loud. Coffee shops are usually busy till 10 with all the "just got to work, time to step out and get a coffee" people.

EDIT: Oops. Missed the part about Polyvore. That works. Baji's as backup.

**Comment by subfuzion** [comment]
I agree @CrabDude ... I had the link for voting, and only threw Specialtys out there as an idea:  http://www.tricider.com/admin/3a9N987Nbtp/3RJNdhjTFs1

I had also considered proposing Clarke's on El Camino because they serve breakfast and it's always easy to get seating, particularly outside. I can actually hold the large table for the whole group.

Baji's also sounds cool...

**Comment by subfuzion** [comment]
For what it's worth, I didn't think it would actually be hard to get to Specialtys early to hold tables in the back area or outside for our group, and it has good food and coffee options for everyone. Along those lines, Boulanger might also be a good choice:  http://www.yelp.com/biz/le-boulanger-mountain-view?osq=boulangerie

**Comment by CrabDude** [comment]
@tonypujals Wait, so which one? Polyvore, Specialty's or Baji's?

**Comment by ghost** [comment]
Thank you. Looking forward to common synergies.
(I'll vote up Baji's if someone puts it out as a choice. )
- chanda

On Tue, Apr 21, 2015 at 1:37 PM, Adam Crabtree notifications@github.com
wrote:

> Welcome @chandadharap https://github.com/chandadharap! Glad to have you
> join us. I hope to see you at breakfast.
> 
> Another option for breakfast is Baji's http://bajis.com in Mountain
> View. They should be dead around that time, and if I let them know, we
> could probably setup a large table in the back-room. Of course, it's a
> little less drop-in friendly, but they're crazy nice, cheap and tasty. Just
> bring cash and I'll provide change & cover the difference.
> 
> My only worry about Specialty's is that it'll probably get pretty loud.
> Coffee shops are usually busy till 10 with all the "just got to work, time
> to step out and get a coffee" people.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/BayNode/nodebreakfast/issues/5#issuecomment-94932286.

## 

Chanda Dharap
Director, Engineering
@StrongLoop Inc
(510) 304-3924
chanda@strongloop.com

_StrongLoop http://strongloop.com/_\* makes it easy to **develop APIs
http://strongloop.com/mobile-application-development/loopback/** in Node,
plus get DevOps capabilities
http://strongloop.com/node-js-performance/strongops/ like monitoring,
debugging and clustering*_._

**Comment by CrabDude** [comment]
With regards to inclusivity and diversity and since this event has traditionally been indirectly BayNode related...

**BayNode has always been and will always be fully inclusive without exception.**

I am blind to the diversity of this group as well as the diversity of BayNode speakers or attendees. I require the same of my organizers. I am uncompromisingly equally supportive and respectful of the concerns of all my event attendees (BayNode, NodeSummit, DallasNode, DallasJS, nodebreakfast, etc...), regardless of any characteristic that may be applied to them. I require the same of my guests.

In this context, inclusiveness merely means this event was invite-only; Now it is not. @emilyrose I would love to have you join us, just as I would equally love to have anyone else here or on Twitter or BayNode join us. I have always enjoyed your presence and contributions when I've been fortunate to be accompany them, and I am equally appreciative of your perspective and input here. Any group demographic trends are a coincidence;  For instance, @ceejbot attended our first Node Enterprise Breakfast at LinkedIn. Diversity of thought and perspective breeds excellence, something all are capable of contributing independent of background.

As an inclusive event, all guests will be joyously welcomed. **If anyone would like to see more genders, ethnicities, nationalities, religions, political views, ages, etc... at this event, you have a responsibility to invite them.**

_With deep respect, I recognize I am contributing to an off-topic sensitive discussion, one that is prone to unintentional and intentional misunderstanding and mischaracterizations. Please accept my sincere apologies in advance for pruning any further discussion on the topic from this conversation (including positive responses). I am always available to discuss this further, including in person. Reach out to me at [BayNode](https://secure.meetup.com/messages/?new_convo=true&member_id=10422423&name=Adam+Crabtree), and I view it as my personal responsibility to ensure the above values._

/cc @emilyrose @chapel @tonypujals.

**Comment by subfuzion** [comment]
So this whole voting thing really isn't working out, is it. lol
http://www.tricider.com/admin/3a9N987Nbtp/3RJNdhjTFs1

Well, if no one is going to vote, then I guess Chanda as our newest member gets the honor of being the decider! :)

Baji's it is!

**Comment by subfuzion** [comment]
Hey all, Greg Stock has generously offered to host breakfast at Polyvore's office [here](https://www.google.com/maps/place/Polyvore/@37.393703,-122.076619,17z/data=!3m1!4b1!4m2!3m1!1s0x808fb7314a9314cb:0x1602ddcf1b45253) in Mountain View this Friday. We settled on Baji's yesterday, but Greg reached out in the afternoon.

What do you all think? If we keep it at Baji's, I'm sure the offer will still be good for the next time. I'm passing the invitation along since it might be convenient with regard to the seating situation. Thoughts from anyone?

**Comment by dshaw** [comment]
+1 for Polyvore. They have a nice space.

**Comment by CrabDude** [comment]
+1 Polyvore sounds good.

On Wednesday, April 22, 2015, Dan Shaw notifications@github.com wrote:

> +1 for Polyvore. They have a nice space.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/BayNode/nodebreakfast/issues/5#issuecomment-95335669.

## 

Adam Crabtree
Noderiety
PO Box 3823
Los Altos, CA 94024
Telephone: _(_469) 777-8415 https://mail.google.com/
Email: dude@noderiety.com

This e-mail message is intended for the use of the addressee(s) only and
may contain information that is privileged-confidential and exempt from
disclosure under applicable law.  If the reader of the message is not the
intended recipient or agent responsible for delivering the message to the
intended recipient, you are hereby notified that any review, dissemination,
distribution or copying of this communication is strictly prohibited.  If
you receive this communication in error, please notify us immediately by
reply e-mail and delete the original message.  Thank you.

**Comment by rosskukulinski** [comment]
+1 works for me.

**Comment by ghost** [comment]
Sounds good.

On Wed, Apr 22, 2015 at 1:55 PM, Tony Pujals notifications@github.com
wrote:

> Hey all, Greg Stock has generously offered to host breakfast at Polyvore's
> office here
> https://www.google.com/maps/place/Polyvore/@37.393703,-122.076619,17z/data=!3m1!4b1!4m2!3m1!1s0x808fb7314a9314cb:0x1602ddcf1b45253
> in Mountain View this Friday. We settled on Baji's yesterday, but Greg
> reached out in the afternoon.
> 
> What do you all think? If we keep it at Baji's, I'm sure the offer will
> still good for the next time. I'm passing the invitation along since it
> might be convenient with regard to the seating situation. Thoughts from
> anyone?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/BayNode/nodebreakfast/issues/5#issuecomment-95332873.

## 

Chanda Dharap
Director, Engineering
@StrongLoop Inc
(510) 304-3924
chanda@strongloop.com

_StrongLoop http://strongloop.com/_\* makes it easy to **develop APIs
http://strongloop.com/mobile-application-development/loopback/** in Node,
plus get DevOps capabilities
http://strongloop.com/node-js-performance/strongops/ like monitoring,
debugging and clustering*_._

**Comment by subfuzion** [comment]
Well I think that settles the venue (finally) ;)  Looks like Polyvore ftw.  Look forward to seeing you all Friday morning!

**Comment by subfuzion** [comment]
If everyone planning on coming can update the checkboxes at the top so Polyvore knows how many to expect, that would be great.

**Comment by subfuzion** [comment]
And just to emphasize, @BayNode/nodebreakfast is now an open group. If there's anyone you want to invite or bring along with you, please do so. We ask that everyone abide by our [code of conduct](https://github.com/BayNode/BayNode/blob/master/code-of-conduct.md). Thanks

**Comment by GregStock** [comment]
We're looking forward to hosting this group and meeting you all.  We have offices on the first and second floors of the building but the breakfast will be up on the second floor.  Take the stairs from the atrium to the top, make your first left and I'll be there to greet you.  Plenty of free 2 hour parking on View Street, W. Evelyn and Villa.  Ping me if you have questions about how to get here.

**Comment by GregStock** [comment]
Greg Palmer and Cindy Chu will be attending from Polyvore.

**Comment by subfuzion** [comment]
Thanks @GregStock :facepunch:

**Comment by FredKSchott** [comment]
Hey all, I won't be able to make it down from SF this morning. Have a great time!

**Comment by CrabDude** [comment]
Looks like I won't be able to make it this morning. Next time hopefully.

**Comment by subfuzion** [comment]
Great to see everyone who made it today. Some great conversations; awesome to hear how different teams are using node and what their engineering processes are like. Look forward to next time. Closing the issue now.

**Comment by aheckmann** [comment]
Was great! looking forward to next time.

—
